src/services/getAppInfo.js

```diff
@@ -1,20 +1,22 @@
 'use strict';
 
 const express = require('express');
+const config = require('../config');
+const pkgJSON = require('../../package.json');
+const gitData = require('../services/getGitData');
 
 const app = express();
 const env = app.get('env');
 
-const pkgJSON = require('../../package.json');
-
-function getAppInfo() {
+async function getAppInfo() {
+  const gitInfo = await gitData();
   return new Promise((resolve) => {
     resolve({
       title: pkgJSON.name,
       environment: env,
       version: pkgJSON.version,
-      commit: 'TODO!',
-      build: 'TODO!',
+      commit: config.herokuSlugCommit || gitInfo.long,
+      build: config.herokuSlugDescription || gitInfo.short,
```

Comment:
Seems this should be either HEROKU_RELEASE_VERSION or some other version increment for the API...

Reply:
@ronwarner1231 It looks like the version is being pulled from package.json